pep-0458.txt

```diff
@@ -163,28 +170,24 @@ Terms used in this PEP are defined as follows:
 * Simple index: The HTML page that contains internal links to the
   distributions of a project [17]_.
 
-* Roles: There is one *root* role in PyPI.  There are multiple roles whose
-  responsibilities are delegated to them directly or indirectly by the *root*
-  role. The term top-level role refers to the *root* role and any role
-  delegated by the *root* role. Each role has a single metadata file that it is
-  trusted to provide.
+* Roles: There is one *root* role in PyPI.  There are multiple other roles for which the *root* role delegates
+  responsibilities, directly or indirectly. The term top-level role refers to the *root* role and any role
+  specified directly by the *root* role, i.e. *timestamp*, *snapshot* and
+  *targets* roles. Each role has a single metadata file that it is trusted to
+  provide.
 
 * Metadata: Metadata are signed files that describe roles, other metadata, and
   target files.
```

Comment:
I think "target file" could stand to be defined here. We usually refer to these as "release files" or "artifacts" when working on PyPI/pip, so drawing the parallel here would be helpful.

Reply:
Thanks for your feedback, @di. I'll add a "target file" entry.

Also, could you please confirm if the terms "projects" , "releases" and "distributions" are defined correctly in that section? They all reference the withdrawn [PEP 426](https://www.python.org/dev/peps/pep-0426/). Is there a more up-to-date resource we should refer to for terminology?

Above all, "distributions" is sometimes used interchangeably with "package" in PEP 458. Does one or the other make more sense in PyPI/pip universe?

Reply:
> Is there a more up-to-date resource we should refer to for terminology?

Yes, https://packaging.python.org/glossary/

> Above all, "distributions" is sometimes used interchangeably with "package" in PEP 458. Does one or the other make more sense in PyPI/pip universe?

Technically a "distribution package" is a type of package. That PEP isn't concerned with other types of packages, so it uses them interchangeably. Since this PEP is concerned with individual files, I think the term "distribution files" makes the most sense to use here.

Reply:
Thanks for the helpful pointer, @di!